scripts/webui.py

```diff
@@ -2099,10 +2124,10 @@ def run_RealESRGAN(image, model_name: str):
 img2img_toggle_defaults = [img2img_toggles[i] for i in img2img_defaults['toggles']]
 img2img_image_mode = 'sketch'
 
-def change_image_editor_mode(choice, cropped_image, resize_mode, width, height):
+def change_image_editor_mode(choice, cropped_image, mask, resize_mode, width, height):
```

Comment:
Note: here was an argument missing, I inserted it. It was no issue though, as the arguments are not used anyway.